test/vapor/config/env_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule Vapor.Config.EnvTest do
+  use ExUnit.Case, async: false
+
+  alias Vapor.Config.Env
+
+  test "with_prefix/1 loads all env variables with a given prefix" do
+    System.put_env("APP_FOO", "foo")
+    System.put_env("APP_BAR", "bar")
+
+    plan = Env.with_prefix("APP")
+    {:ok, envs} = Vapor.Provider.load(plan)
+    assert envs["foo"] == "foo"
+    assert envs["bar"] == "bar"
+  end
+
+  test "with_bindings/1 loads any env variables it can find" do
+    System.put_env("FOO", "foo")
+    System.put_env("BAR", "bar")
+
+    plan = Env.with_bindings([
+      foo: "FOO",
```

Comment:
Oh! These are _translation_ bindings. Sorry, it took me a hot minute. What do you think about making that clearer by giving them different names? 
```suggestion
      foo: "GOO",
```